test/wireFormatGeneratorTests.ts

```diff
@@ -1,24 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-import * as glob from "glob"
-import * as jsYaml from "js-yaml"
-import * as path from "path"
-// tslint:disable-next-line: no-implicit-dependencies
-import * as should from "should"
-
-const yamlPaths = glob.sync(path.join(__dirname, "swaggers/**/yaml/*.yml"))
```

Comment:
afaict no files match this glob so no tests were running for this file

Reply:
@vladbarosan can you confirm this is fine or tell me what files this should be run on please?

Reply:
@billytrend no idea what the wireFormatGenerator is @sergey-shandar  you know ?